packages/ui/src/components/EventsCard/EventsCard.test.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Event, EventsCard } from './EventsCard';
+import { BUTTON_TEXT, EVENTS as listOfEvents, MAIN_TITLE } from './EventsCard.mocks';
+
+describe('EventsCard tests', () => {
```

Comment:
🤔 It looks to me, that we're trying to test too many things here. In my opinion, you can create only one test to check if events render correctly and two tests to check date display correctness (for a single date and for the time period)

Reply:
Will be 7 tests, one for main title, one for properly render events, one for properly text in buttton, two for date (a single date and for the time period) and two for hours (single date and for the time period).